Python/Tests/Utilities.Python/PythonTestExplorer.cs

```diff
@@ -32,9 +33,10 @@ private static class TestCommands {
             public const string CopyDetails = "TestExplorer.CopyDetails";
         }
 
-        public PythonTestExplorer(VisualStudioApp app, AutomationElement element)
+        public PythonTestExplorer(VisualStudioApp app, AutomationElement element, AutomationElement searchElement)
             : base(element) {
             _app = app;
+            _searchBarElement = searchElement ?? throw new ArgumentNullException(nameof(searchElement));
```

Comment:
store this as an `AutomationWrapper` (ctor takes in an `AutomationElement`)
see later comment for why